src/NServiceBus.Core/obsoletes.cs

```diff
@@ -1446,16 +1446,29 @@ public class WellKnownStep
         public static readonly WellKnownStep EnforceUnsubscribeBestPractices;
     }
 
-    public partial class StepRegistrationSequence
+    [ObsoleteEx(
+            RemoveInVersion = "7.0",
+            TreatAsErrorFromVersion = "6.0",
+            Message = "Use the 'PipelineSettings' class to register behaviors instead.")]
+    public class StepRegistrationSequence
```

Comment:
hm you're right. I thought I checked that but seems like I didn't do that properly then. Will just remove the class.